openedx/features/course_experience/urls.py

```diff
@@ -63,4 +64,9 @@
         dismiss_welcome_message,
         name='openedx.course_experience.dismiss_welcome_message',
     ),
+    url(
+        r'^dates_fragment$',
+        CourseDatesFragmentMobileView.as_view(),
```

Comment:
I'd rather this expose the standard ``CourseDatesFragmentView`` at this URL.

Reply:
@andy-armstrong I thought same but mobile team want 404 for unauthenticated users.

Reply:
Can you change the URL to be ``r'^mobile_dates_fragment$'``, as this isn't exposing the dates fragment.

Reply:
Sure, thanks!